### DISCUSSION TITLE
Unable to Update AdGuard Home

### DISCUSSION THREAD

**Discussion by noobdesigner922** [AdguardTeam/AdGuardHome#5141] [Q&A]
I installed AdGuard Home on Ubuntu 20.04.5 LTS on Oracle Cloud Free Tier VPS with their Arm-based Ampere A1 cores by following [this video guide](https://youtu.be/eteKnorz4LQ). I have been using it for a couple of months now. I successfully installed updates till AGH version v0.107.14, but when I try to update to v0.107.18, the update fails, and AGH asks me to update manually. When I follow the guide mentioned [here](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#manual-update) However, I keep getting this error message.

`tar (child): /home/ubuntu/Downloads/AdGuardHome linux arm64.tar.qz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now`

What am I doing wrong, and how do I successfully update?

![Screenshot 2022-11-13 at 12 40 16 PM](https://user-images.githubusercontent.com/72648637/201511917-a4346063-efaa-4da3-9163-6b5ac10221b6.png)
<img width="372" alt="Screenshot 2022-11-13 at 1 07 39 PM" src="https://user-images.githubusercontent.com/72648637/201511921-d077e0c3-14d1-4b7e-abdd-48b181c99377.png">

**Comment by andekande** [comment]
You downloaded to /tmp/ and want to extract from ~/Downloads
You probably also want to use the amd64 and no arm64 build, depending on your hardware.
Becaus untar didn't work, no update took place.
